docs/mine-hnt/validators/validator-deployment-guide.mdx

```diff
@@ -0,0 +1,402 @@
+
+And thanks to the `--mount option`, the blockchain data and the validator keys are preserved through updates.
+
+:::info
+
+Now that you have your Validator running, you need to [actually Stake tokens to make it official.](#stake-tokens-to-your-validator)
+
+:::
+
+
+### Deploy Miner from Source 
+
+Ok, brave soul. Here's what you'll need to build and deploy the miner from source:
+
+
+* miner
+* Erlang package (the latest 22 or 23 versions are supported, the 24 release candidate will NOT work)
+* `libwxgtk` package
+* git 
+
+
+Before we get started, make sure you're fully up to date:
+
+```
+sudo apt-get update -y
+```
+
+Next, let's install all the things. Start by cloning the `validator` branch of `miner`:
+
+```
+git clone -b pevm/validators https://github.com/helium/miner.git
+```
+
+Then proceed to get you some Erlang (here we're specifying `22.3.1`):
+
+```
+wget https://packages.erlang-solutions.com/erlang/debian/pool/esl-erlang_22.3.1-1~ubuntu~bionic_amd64.deb
+```
+
+Next, get the `libwxgtk` package:
+
+```
+wget http://mirrors.kernel.org/ubuntu/pool/universe/w/wxwidgets3.0/libwxgtk3.0-0v5_3.0.4+dfsg-3_amd64.deb
+```
+
+And finish thins off by acquiring a some wonderful dependencies:
+
+```
+sudo apt install -y libdbus-1-dev autoconf automake libtool flex libgmp-dev cmake libsodium-dev libssl-dev bison libsnappy-dev libclang-dev doxygen make cargo
+```
+
+```
+sudo apt install g++ -y
+sudo apt install -y libsctp1
+sudo apt install -y libncurses5
+sudo apt install -y libwxbase3.0-0v5
+sudo apt install -y build-essential cmake
+sudo apt install -y libdbus-1-dev
+sudo apt install -y mosh vim 
+sudo apt install -y parallel
+```
+
+Then we need to unpack and install all of this stuff. Start with `libwxgtk`:
+
+```
+sudo dpkg -i libwxgtk3.0-0v5_3.0.4+dfsg-3_amd64.deb
+sudo apt update -y
+```
+
+Then onto Erlang. First `unpack` then `install`:
+
+```
+sudo dpkg -i esl-erlang_22.3.1-1~ubuntu~bionic_amd64.deb
+```
+
+```
+sudo apt-get install -f
+```
+
+Then navigate to your `miner` directory:
+
+```
+cd miner
+```
+
+And, finally, build the `miner`
+
+```
+./rebar3 as validator release
+```
+
+#### Load the Genesis Block
+
+One last step. When building from source, you'll need to manually load the genesis block once your miner is deployed and running. To do this:
+
+```
+wget https://snapshots.helium.wtf/genesis.testnet
+miner genesis load </absolute/path/to/genesis/block>
+```
+
+## Stake Tokens to Your Validator
+
+Now that your Validator node is running, the final step in the process is to formally stake TNT to your Validator. As part of the staking process the Validator address needs to both be in the staking transaction and sign the transaction. After a wallet stakes a validator node, the wallet becomes that node’s owner, has control over that validator node, and receives rewards.
+
+First, double check your wallet balance to make sure you have the `10000` TNT required:
```

Comment:
you'll also need enough TNT to cover the fee, which the extra 5 TNT from the faucet easily covers

Reply:
Ack